zerver/actions/message_edit.py

```diff
@@ -119,6 +119,31 @@ def validate_message_edit_payload(
         raise JsonableError(_("Widgets cannot be edited."))
 
 
+def validate_user_message_edit(
```

Comment:
`validate_message_content_edit` would be more correct -- it's checking if they're allowed to edit the message content.

Reply:
ok. I'll update it.